src/main/java/book/store/dto/request/CreateBookRequestDto.java

```diff
@@ -0,0 +1,21 @@
+package book.store.dto.request;
+
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+public class CreateBookRequestDto {
+    @NotNull
+    private String title;
+    @NotNull
+    private String author;
+    @NotNull
+    private String isbn;
+    @NotNull
```

Comment:
never use annotations if you dont understand how they work



```suggestion
    private String title;
    private String author;
    private String isbn;
```

Reply:
@sokimaaa 
I am using this annotation for receiving not null values from requests to the server in the BookController:
@PostMapping("/books")
    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
        return bookService.createBook(requestDto);
    }
Is this solution doesn't correct?